ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -604,3 +604,72 @@ literals:
 
 # Added 4/12/20
 - supply_air_damper_percentage_sensor
+
+# Added 9/8/20 for SG-SIN-MBC2
+- chilled_return_water_flowrate_sensor
+- device_fault_status
+- fire_alarm_status
+- fire_fault_status
+- makeup_water_high_tank_percentage_status
+- makeup_water_low_tank_percentage_status
+- max_cooling_air_flowrate_setpoint
+- min_cooling_air_flowrate_setpoint
+- pump_fault_status
+- pump_run_status
+- recovery_alarm_command
```

Comment:
what does an alarm command entail? are you commanding it to alarm?

Reply:
it is a "fail to start reset command". It is a command to reset an alarm

Reply:
two questions: 
first, is this point needed? what is the intended use case?
second, wouldnt the alarm just reset when the alarm condition goes away?
third, if it really is a reset for a particular alarm, it should have similar structure to the field it is resetting; for example, if its `supply_fan_alarm_status`, the alarm reset should be something like `reset_supply_fan_alarm_command`, so its clear what alarm is resetting.